frontend/components/navigation/Tabs/TabHeader.tsx

```diff
@@ -35,16 +39,25 @@ const getIcon = {
 export const TabHeader = ({
 	name,
 	style,
+	notificationBadgeContent = "",
 	className = "",
 	...props
 }: TabHeaderProps) => {
 	return (
 		<div
-			className={`flex flex-row items-center space-x-1 ${className}`}
+			className={`flex flex-row items-center space-x-2 ${className}`}
 			{...props}
 		>
 			{getIcon[style]}
 			<div>{name}</div>
+			{notificationBadgeContent && (
+				<div className="relative flex h-full text-sm text-eggshell">
+					<div className="absolute inline-flex h-full w-full animate-ping rounded-full bg-kiokuRed opacity-75"></div>
```

Comment:
```suggestion
					<div className="absolute inline-flex h-full w-full animate-[ping_1s_ease-in-out_5] rounded-full bg-kiokuRed opacity-75"></div>
```
I don't know if I am a fan of the infinite ping of the badge. Maybe we can set a limit for it, the suggestion is just an idea.

Reply:
fixed in 1163621cf64ae16eb60667af9a6c96e31a512f8d